pkg/trace/config/config.go

```diff
@@ -588,3 +598,9 @@ func (c *AgentConfig) AllFeatures() []string {
 	}
 	return feats
 }
+
+func InAzureAppServices(lookupEnv func(string) (string, bool)) bool {
+	_, existsLinux := lookupEnv(RunZip)
+	_, existsWin := lookupEnv(AppLogsTrace)
```

Comment:
I'll usually avoid creating higher order functions for simple checks like this. Let's have `InAzureAppServices()` take no parameters and make use of the `os` package directly: `_, existsLinux := os.LookupEnv(RunZip)`

Reply:
I passed it in here so I could "mock" the behavior in tests like the one below for example. Is there a better way to mock or could this test be removed altogether?

```
func TestInLinuxAzureAppServices(t *testing.T) {
	isLinuxAzure := InAzureAppServices(func(s string) (string, bool) { return "", true })
	assert.True(t, isLinuxAzure)
}

Reply:
Ah I see where you're coming from. Yup, you can just use `os.Setenv(RunZip, "✌️")` in your unit tests